src/utils/ride_status.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+class RideStatus(Enum):
+    NONE = "None"
+    REQUESTED = "Requested"
+    IN_PROGRESS = "In Progress"
```

Comment:
We have 2 phases of IN_PROGRESS which are "PICKING_UP" and "IN_TRIP". Could you separate them?